api/java/org/openyolo/api/CredentialClient.java

```diff
@@ -272,7 +296,15 @@ public CredentialRetrieveResult getCredentialRetrieveResult(
         }
 
         try {
-            return CredentialRetrieveResult.fromProtobufBytes(resultBytes);
+            CredentialRetrieveResult result =
+                    CredentialRetrieveResult.fromProtobufBytes(resultBytes);
+
+            // Once a successfully retrieve result has been handled, re-enable auto sign-in.
```

Comment:
We may not even need to be this specific; disabling for a single request as long as we get _any_ valid response may be sufficient.